Yoast/Sniffs/WP/RequiredParametersSniff.php

```diff
@@ -0,0 +1,91 @@
+<?php
+/**
+ * YoastCS\Yoast\Sniffs\Files\FileNameSniff.
+ *
+ * @package Yoast\YoastCS
+ * @author  Jip Moors
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+namespace YoastCS\Yoast\Sniffs\WP;
+
+use WordPress\AbstractFunctionParameterSniff;
+
+/**
+ * Verifies that optional parameters are set, when they should be set.
+ *
+ * @package Yoast\YoastCS
+ * @author  Jip Moors
+ *
+ * @since   0.6
+ */
+class RequiredParametersSniff extends AbstractFunctionParameterSniff {
```

Comment:
Just wondering - would `RequiredOptionalParameters` better describe what the sniff covers ?